spec/integration/data_stream/manifest.spec.yml

```diff
@@ -83,6 +83,12 @@ spec:
             default: false
             examples:
               - true
+          secret:
```

Comment:
Not sure about removing this field in previous versions using a JSON Patch.
If we do so, probably it should be for 2.9.0 (and it would be needed to update changelog too) ? Would it be better to keep for all versions for security reasons ?  @jsoriano  WDYT ?

```yaml
versions:
  - before: 2.9.0 # other version 2.0.0 ? 
    patch:
      - op: remove
        path: /definitions/vars/items/properties/secret
```

Reply:
+1 to add this to encourage updating the spec (and to update the version in the changelog).